src/testdrive/src/action/skip_if.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use anyhow::Context as _;
+use async_trait::async_trait;
+
+use crate::action::{Action, Context, State};
+use crate::parser::BuiltinCommand;
+
+pub struct SkipIfAction {
+    query: String,
+    context: Context,
+}
+
+pub fn build_publish(
+    mut cmd: BuiltinCommand,
+    context: Context,
+) -> Result<SkipIfAction, anyhow::Error> {
+    let query = cmd.args.string("sql")?;
```

Comment:
Probably just use `cmd.input` here? The SQL string is much easier to deal with as "input" (i.e., stuff you type on the next line) rather than an argument.